tests/test_edge_population.py

```diff
@@ -728,30 +728,13 @@ def test_iter_connection_unique(self):
     def test_h5_filepath_from_config(self):
         assert self.test_obj.h5_filepath == str(TEST_DATA_DIR / "edges.h5")
 
-    @pytest.mark.skip(reason="Until spatial-index is released publicly")
-    def test_spatial_synapse_index(self):
```

Comment:
why delete this one?

Reply:
This is the first population that does not define the spatial index directory.

Besides, at the end of the file, this test is done for the second edge population with a defined spatial index path, and done properly.

(the call will contain an absolute path, and not just the final fragment)